src/firstboot/firstboot.c

```diff
@@ -127,7 +127,7 @@ static void print_welcome(int rfd) {
         else
                 printf("\nWelcome to your new installation of %s!\n", pn);
 
-        printf("\nPlease configure your system!\n\n");
+        printf("\nPlease configure your system!\n");
```

Comment:
Wouldn't it make sense to colorize this as well?

Reply:
maybe, dunno. i was thinking we should eventually color the bg of these firstboot wizards in blue color, in reminiscence to the good old windows installer, but maybe let's leave that for another time.